packages/api/src/storage.ts

```diff
@@ -79,7 +80,10 @@ export default class Storage {
   }
 
   static getDefaultLockfile(): Lockfile {
+    const majorVersion = semver.parse(PACKAGE_VERSION)?.major || 'latest';
+
     return {
+      $schema: `https://unpkg.com/api@${majorVersion}/schema.json`,
```

Comment:
I'm using a major version here because I don't really want the schema URL to change with every minor and patch release we push out. However using a major here also means that until we tag a v7 this URL isn't going to work as `api@7` doesn't match `api@7.0.0-beta.0`.

For example see that https://unpkg.com/api@7.0.0-beta.0/package.json works but https://unpkg.com/api@7/package.json does not.

I think I'm okay with this? I don't really want to have code in here to detect if `PACKAGE_VERSION` is a beta or alpha and use the full version.

Reply:
yeah agreed, esp since we're capturing exact versions in the `installerVersion` value!

Reply:
actually wait — since we technically version our `api.json` independently of our `api` package, shouldn't it point to something like `https://unpkg.com/api@schema-v1/package.json`? thoughts on maintaining a `dist-tag` for that?

Reply:
its technically versioned independently of `api` but we'd only ever ship changes to it with an `api` release so they're kind of tied to each other. i feel like unless a `schema-v1` dist-tag process were built into our lerna config we'd quickly forget to update it.

Reply:
or it could be released as its own package but that feels like a lot of overhead for a json file

Reply:
talked offline and we'll stick with the `api` major version tags as our reference!